### PR TITLE
include ADD signals

### PR DESCRIPTION
**Pull request by fanfanman**
After processing ADD MiniAOD files with the package Zprime2muAnalysis, we have to move ADD root files from folder ADD2016MC/ to the folder files/ in this package.

Then, in order to plot the invariant mass spectrum of DY + ADD signal:
> python plot.py -r -d -p massPlotEle -a -2016 -s Lam6000

The programs used to obtain ADD signal yields are all copied and renamed to avoid conflict:
> ./runADDFits2016.sh
> python ADDsignalYields.py
The resulting pkl files are stored in folder ADDpkl/

### CODE REVIEW

File: plot.py [hunk 1]
```diff
@@ -54,7 +56,7 @@ def plotDataMC(args,plot):
 	for background in backgrounds:
 		if args.use2016:
 			if background == "Jets":
-				processes.append(Process(getattr(Backgrounds2016,background),eventCounts,negWeights,normalized=True))
+				processes.append(Process(getattr(Backgrounds,background),eventCounts,negWeights,normalized=True))
```
Comment (JanFSchulte) [inline plot.py:59]: Why did you change this?

File: plot.py [hunk 2]
```diff
@@ -330,14 +329,14 @@ def plotDataMC(args,plot):
 	if args.ratio:
 
 		ratioPad.RedrawAxis()
-	if not os.path.exists("plots"):
-		os.makedirs("plots")	
+	if not os.path.exists("test"):
```
Comment (JanFSchulte) [inline plot.py]: This doesn't look that useful to me, why not keep "plots"?

File: .gitignore [hunk 3]
```diff
@@ -2,3 +2,4 @@
 *.root
 *json
 *.txt
+files/*.root
```
Comment (JanFSchulte) [inline .gitignore:6]: can you please add .pdf files to the .gitignore and remove all the PDFs added to the repository?

File: ADDfitUtils.py [hunk 4]
```diff
@@ -0,0 +1,287 @@
+def doFitOnGraph(params, lvals, xvals, xerrs,
```
Comment (JanFSchulte) [inline ADDfitUtils.py]: Is it necessary to duplicate all this code and all the files related to the fit? I would prefer if you could find a why to integrate the changes necessary for the ADD and make it configurable. Less code to maintain is always good. :)

File: plot.py (hunk 1 above)
Comment (fanfanman) [reply plot.py:59]: When I used Backgrounds2016, the 2016 Jets background is not showing....

File: plot.py (hunk 2 above)
Comment (fanfanman) [reply plot.py]: sorry my bad, will change this back to plots.

File: plot.py (hunk 1 above)
Comment (JanFSchulte) [reply plot.py:59]: Mhh, ok. But we should try to understand and fix that properly. Can you dig a little bit more? If you have trouble finding the source of the issue, I can also have a look.

Comment (fanfanman) [reply plot.py:59]: Sure. I just digged a little bit, the jets 2016 background is read from file jets_muons_2016.root, which ranges from 0.5-3000.5 GeV. When we're trying to load jets_2016 background, the program will first call loadHistoFromFile at (https://github.com/JanFSchulte/CIAnalysis/blob/master/helpers.py#L274), then this function will load the histogram ("jets") from file ("jets_muons_2016.root") which is correct, but then it will try to rebin it at (https://github.com/JanFSchulte/CIAnalysis/blob/master/helpers.py#L106) with a new binning from (https://github.com/JanFSchulte/CIAnalysis/blob/master/helpers.py#L42). However this new binning ranges from 51 to 4000 which exceeds the boundary.... So i think that's why it doesn't work... In contrast, the 2017 jets background has the same structure but it ranges from 0-7000, so it can successfully pass the binning.
